test/resources/MainnetAddresses.sol

```diff
@@ -175,6 +180,62 @@ contract MainnetAddresses {
     address public aave3Pool = 0xDeBF20617708857ebe4F679508E7b7863a8A8EeE;
     ERC20 public CRV_AAVE_3CRV = ERC20(0xFd2a8fA60Abd58Efe3EeE34dd494cD491dC14900);
     address public stETHWethNg = 0x21E27a5E5513D6e65C4f830167390997aA84843a;
+    address public EthFrxEthCurvePool = 0xa1F8A6807c402E4A15ef4EBa36528A3FED24E577;
```

Comment:
Seeking clarity on my understanding with this: **QUESTION: why is coins[0] required to be supported for all of this?**

Context / my understanding of things:
```

// EMA Pricing Pools
    /**
     * I thought that there was either EMA pricing or 2Pool pricing. 
     * I think it goes:
     * - Set up 2Pool Pricing for a pool: coins[0] needs to be supported or else things revert. This is where Crispy ended up using an EMA for mkUSD with another curvePool to price it out.
     * - Setup cont.: isCorrelated means we do not use lp_price(), we instead just have to get pricing of coins[1] too, and make sure it is supported. --> it uses both coins[i] to get the price in USD using stored rates for the respective pricing extension.
     * - Setup cont.: If it is not correlated, it tries lp_price().
     * 
     * CURVE EMA:
     * - This is just pricing using curve pools for a respective asset. It REQUIRES having pricing supported for the coins[0] asset. Then it queries: pool.price_oracle(index) or pool.price_oracle() to get the price which is factors by stored_rates()[index] if it has handleRate as true.
     * 
     * 
     */
```